windows_containers/windows-containers-release-notes-3-x.adoc

```diff
@@ -35,6 +35,7 @@ Running Windows container workloads is not supported for clusters in a restricte
 ====
 
 Version 3.x of the WMCO is only compatible with {product-title} 4.8.
+//Is this still true?
```

Comment:
@EricPonvelle, PTAL?

Reply:
@kalexand-rh yes, this is true. A new version of the WMCO release notes will be merged for OCP 4.9. WMCO 3.x only applies to OCP 4.8. WMCO 4.x will apply to OCP 4.9.

Reply:
Thank you for confirming!